server/src/main/java/io/seata/server/session/DefaultSessionManager.java

```diff
@@ -51,6 +51,22 @@ public DefaultSessionManager(String name) {
             public boolean writeSession(LogOperation logOperation, SessionStorable session) {
                 return true;
             }
+
+            @Override
+            public long getCurrentMaxSessionId() {
+                long maxSessionId = 0L;
+                for (Map.Entry<String, GlobalSession> entry : sessionMap.entrySet()) {
+                    GlobalSession globalSession = entry.getValue();
+                    if (globalSession.hasBranch()) {
+                        long maxBranchId = globalSession.getSortedBranches().get(globalSession.getSortedBranches().size() - 1)
```

Comment:
getSortedBranches seems not implement?

Reply:
Because the order is self-increasing the branchId, the added branchId is incremented.